tests/Report/Reporter/GitlabReporterTest.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TwigCsFixer\Tests\Report\Reporter;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Output\BufferedOutput;
+use Symfony\Component\Console\Output\OutputInterface;
+use TwigCsFixer\Report\Report;
+use TwigCsFixer\Report\Reporter\GitlabReporter;
+use TwigCsFixer\Report\Violation;
+use TwigCsFixer\Report\ViolationId;
+use TwigCsFixer\Test\TestHelper;
+
+final class GitlabReporterTest extends TestCase
+{
+    public function testGetName(): void
+    {
+        static::assertSame(GitlabReporter::NAME, (new GitlabReporter())->getName());
+    }
+
+    /**
+     * @dataProvider displayDataProvider
+     */
+    public function testDisplayErrors(string $expected, ?string $level, bool $debug): void
+    {
+        $textFormatter = new GitlabReporter();
+
+        $file = TestHelper::getOsPath('tests/Report/Reporter/Fixtures/file.twig');
+        $report = new Report([new \SplFileInfo($file)]);
+
+        $violation0 = new Violation(
+            Violation::LEVEL_NOTICE,
+            'Notice',
+            $file,
+            'Rule',
+            new ViolationId('NoticeId', null, 1)
+        );
+        $report->addViolation($violation0);
+        $violation1 = new Violation(
+            Violation::LEVEL_WARNING,
+            'Warning',
+            $file,
+            'Rule',
+            new ViolationId('WarningId', null, 2, 22)
+        );
+        $report->addViolation($violation1);
+        $violation2 = new Violation(
+            Violation::LEVEL_ERROR,
+            'Error',
+            $file,
+            'Rule',
+            new ViolationId('ErrorId', null, 3, 33)
+        );
+        $report->addViolation($violation2);
+        $violation3 = new Violation(
+            Violation::LEVEL_FATAL,
+            'Fatal'."\n".'with new line',
+            $file,
+            'Rule',
+            new ViolationId('FatalId')
+        );
+        $report->addViolation($violation3);
+
+        $output = new BufferedOutput(OutputInterface::VERBOSITY_NORMAL, true);
+        $textFormatter->display($output, $report, $level, $debug);
+
+        $text = $output->fetch();
+
+        $actual = json_decode($text);
+        $actual = json_encode($actual, \JSON_PRETTY_PRINT | \JSON_UNESCAPED_SLASHES);
+
+        static::assertSame($expected, $actual);
```

Comment:
I'm not sure how to fix the issue for the Window build.

Maybe you could compare the json instead with `assertJsonStringEqualsJsonString` ?